i18n/en.json

```diff
@@ -6136,12 +6136,16 @@
     "translation": "An error occurred while saving the file to {{.Filename}}"
   },
   {
+    "id": "utils.config.supported_available_locales.app_error",
+    "translation": "Unable to load mattermost configuration file:  AvailableLocales must include all of the supported locales. Set AvailableLocales to empty string as default value."
```

Comment:
Just a minor proposal: `must include all of the supported locales` --> `must include DefaultClientLocale`

`Unable to load mattermost configuration file:  AvailableLocales must include DefaultClientLocale. Set AvailableLocales to empty string as default value.`